src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -395,21 +384,22 @@ def test_WINNF_FT_S_FAD_1(self, config_filename):
         # check GeoJson Winding of PPA record
         utils.HasCorrectGeoJsonWinding(ppa_record['zone']['features'][0]['geometry'])     
         exist_in_dump = False
-        for ppa in config['ppaRecords']:
+        for ppaConf in config['ppaRecords']:
+          ppa = ppaConf['ppaRecord']
           if 'id' in ppa:
             del ppa['id'] 
-            # we exclude the values of the CBSD reference Id, because they are not necessarily the same
-            ppa_cbsd_reference_ids = ppa['cbsdReferenceId']
-            ppa_record_cbsd_reference_ids = ppa_record['cbsdReferenceId']
-            del ppa['cbsdReferenceId']
-            del ppa_record['cbsdReferenceId']
-            exist_in_dump = exist_in_dump or areTwoPpasEqual(ppa_record, ppa)
-            # check that the record has the same length of the CBSD reference Ids
-            exist_in_dump &= len(ppa_cbsd_reference_ids) == len(ppa_record_cbsd_reference_ids)
-            ppa['cbsdReferenceId'] = ppa_cbsd_reference_ids
-            ppa_record['cbsdReferenceId'] = ppa_record_cbsd_reference_ids
-          if exist_in_dump:
-            break
+          # we exclude the values of the CBSD reference Id, because they are not necessarily the same
```

Comment:
See my other comment on this: I think it is actually possible to check these values here.